test_resources/serialization_baseline/databases/My Database/My Database.yaml

```diff
@@ -11,8 +11,9 @@ is_full_sync: true
 is_on_demand: false
 is_sample: false
 is_audit: false
-metadata_sync_schedule: 0 17 * * * ? *
-cache_field_values_schedule: 0 0 18 * * ? *
```

Comment:
These fields get reset every time we import, so will change when the "update baseline" REPL function is run.

I was too lazy to back them out :-)